### PR TITLE
allow to require gem as: require 'acme/client'

### PR DESCRIPTION
**Pull request by porbas**
it's kind of standard according to http://guides.rubygems.org/name-your-gem/

### CODE REVIEW

File: lib/acme-client.rb [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# This file lives here for backward compatibility only and should be deprecated soon
+# gem named acme-client should be required as
```
Comment (jhass) [inline lib/acme-client.rb]: It should stay forever too, iirc for example bundler autorequire will still just try plain gem name.

Comment (porbas) [inline lib/acme-client.rb]: Bundler "knows" about namespaced libs and would autorequire acme/client without complaints if there wouldn't be acme-client.rb file.
https://github.com/bundler/bundler/blob/master/lib/bundler/runtime.rb#L97

Comment (jhass) [inline lib/acme-client.rb]: TIL, thanks. I still think it can't harm and it seems to be common practice to offer both.

**Comment by unixcharles** [comment]
👍

**Pull request merged**